src/help.sh

```diff
@@ -38,21 +39,16 @@ function kworkflow_help()
 # installed to the system
 function kworkflow_man()
 {
+  feature="$1"
```

Comment:
fwiu, `feature` is a path and you need to check it. Otherwise, we will get errors like this one:

```
kw man lala
man: /home/siqueira/.local/share/man/lala.1: No such file or directory
```